### ISSUE TITLE
WCAG 1.3.1,WCAG 4.1.2: Ensures select element has an accessible name (#type_doc)

### ISSUE THREAD

**Issue by federicoaposada** [federicoaposada/CUNA#1]
#### Issue
`Ensures select element has an accessible name` ([select-name](https://dequeuniversity.com/rules/axe/4.4/select-name?application=msftAI))
#### Target application
[Constancia de CUIL | anses](https://www.anses.gob.ar/consulta/constancia-de-cuil)
#### Element path
#type_doc
#### Snippet
`<select id="type_doc" name="type_doc" data-drupal-selector="edit-type-doc" class="form-select form-control"><option value="0029">Documento Único</option><option value="0025">Libreta de Enrolamiento</option><option value="0026">Libreta Cívica</option><op...`
#### How to fix
    Fix any of the following:
      Form element does not have an implicit (wrapped) <label>
      Form element does not have an explicit
